llmfoundry/models/hf/hf_fsdp.py

```diff
@@ -229,10 +233,10 @@ def prepare_hf_enc_dec_model_for_fsdp(
     model.shared == model.encoder.embed_tokens and model.shared ==
     model.decoder.embed_tokens
     """
-    tied_embeddings = model.get_input_embeddings()
-    encoder = model.get_encoder()
-    decoder = model.get_decoder()
-    lm_head = model.get_output_embeddings()
+    tied_embeddings = model.get_input_embeddings()  # type: ignore
```

Comment:
lol that's annoying that PeftModel doesn't inherit from PretrainedModel

Reply:
It just mirrors all the attrs via overriding --getattr--